packages/docusaurus/src/webpack/utils.ts

```diff
@@ -559,14 +559,14 @@ export function getMinimizer(
       // Using the array syntax to add 2 minimizers
       // see https://github.com/webpack-contrib/css-minimizer-webpack-plugin#array
       new CssMinimizerPlugin({
+        // @ts-expect-error: the lib def looks terrible...
```

Comment:
what is the reason for code change here?

Reply:
This I have little idea. The typechecking is now reporting errors in different places